soda-data-quality-checks/Makefile

```diff
@@ -0,0 +1,17 @@
+usage:		## Shows usage for this Makefile
+	@cat Makefile | grep -E '^[a-zA-Z_-]+:.*?## .*$$' | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}'
+
+install:    ## Install dependencies
+	which awslocal || pip install 'awscli-local[ver1]'
+	which soda || pip install soda-core-snowflake
+
+init:      ## Initialize the environment and seed test data into local Snowflake
+	soda test-connection -d local_snowflake -c configuration.yml
+	snow sql -c local --query "CREATE TABLE test_table (id INT, name TEXT)"
```

Comment:
I tried running `make init`, connection is established with snowflake, but the `snow` command cannot be found. Full error:

```
soda test-connection -d local_snowflake -c configuration.yml
[17:58:54] Soda Core 3.3.13
Successfully connected to 'local_snowflake'.
Connection 'local_snowflake' is valid.
snow sql -c local --query "CREATE TABLE test_table (id INT, name TEXT)"
/bin/sh: snow: command not found
make: *** [init] Error 127
```

Reply:
Update: managed to configure connection manually, do not know if docs regarding configuring snowflake cli credentials should be covered here.

Reply:
Update: managed to configure connection manually, do not know if docs regarding configuring snowflake cli credentials should be covered here.

Reply:
Thanks, great point - will add a note for that 👍